components/tools/OmeroJava/test/integration/AbstractServerTest.java

```diff
@@ -285,6 +291,143 @@ public void tearDown() throws Exception {
     }
 
     /**
+     * Add the given annotation to the given image.
+     * @param image an image
+     * @param annotation an annotation
+     * @return the new loaded link from the image to the annotation
+     * @throws ServerError unexpected
```

Comment:
The error may or may not be unexpected, it depends on the context of the test's use of the method, so maybe just say you're passing along any error from trying to save the link or something.